pywbem/_server.py

```diff
@@ -1024,6 +1029,44 @@ class variable.
         self._namespaces = [inst['Name'] for inst in ns_insts]
         self._namespace_paths = [inst.path for inst in ns_insts]
 
+        # An old version of a Hitachi server does not return its Interop
+        # namespace in the set of namespace instances. Pywbem adds the
+        # interop namespace to make up for that.
+        if self.interop_ns not in self._namespaces:
+            warnings.warn(
+                _format("Server at {0} has an Interop namespace {1!A}, but "
+                        "does not return it when enumerating class {2!A} "
+                        "- adding it",
+                        self.conn.url, self.interop_ns, ns_classname),
+                ServerWarning, stacklevel=2)
+            self._namespaces.append(self.interop_ns)
+
+            if not self._namespace_paths:
+                warnings.warn(
+                    _format("Server at {0} does not have any namespaces "
+                            "besides the Interop namespace {1!A}",
+                            self.conn.url, self.interop_ns),
+                    ServerWarning, stacklevel=2)
+            else:
+                # Because the different classes used to represent namespaces
+                # have different keys, it seems best to copy one of them.
+                org_path = self._namespace_paths[0]
+                interop_path = org_path.copy()
+                interop_path['Name'] = self.interop_ns
+                try:
+                    self._conn.GetInstance(
```

Comment:
The namespace input parameter should not be there.  In this case, I believe that the namespace is in the instance_path.